packages/image_picker/image_picker_platform_interface/pubspec.yaml

```diff
@@ -3,21 +3,21 @@ description: A common platform interface for the image_picker plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.1.6
+version: 2.0.0-nullsafety
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.1.8
-  http: ^0.12.1
-  plugin_platform_interface: ^1.0.2
+  meta: ^1.3.0-nullsafety.6
+  http: ^0.13.0-nullsafety.0
+  plugin_platform_interface: ^1.1.0-nullsafety.2
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  mockito: ^4.1.1
-  pedantic: ^1.8.0+1
+  mockito: ^5.0.0-nullsafety.5
```

Comment:
I checked, and mockito doesn't seem to be used in this package. It can probably be removed :)

Reply:
done